cmd/sql/flow.go

```diff
@@ -4,273 +4,235 @@ import (
 	"fmt"
 	"os"
 	"path/filepath"
-	"strings"
 
 	"github.com/astronomer/astro-cli/sql"
 	"github.com/spf13/cobra"
 )
 
+// All cmd names
+const (
+	aboutCmdName    = "about"
+	configCmdName   = "config"
+	generateCmdName = "generate"
+	initCmdName     = "init"
+	runCmdName      = "run"
+	validateCmdName = "validate"
+	versionCmdName  = "version"
+)
+
+// All cmd flags
 var (
-	environment       string
-	connection        string
-	airflowHome       string
 	airflowDagsFolder string
-	projectDir        string
+	airflowHome       string
+	connection        string
+	debug             bool
+	env               string
 	generateTasks     bool
+	key               string
 	noGenerateTasks   bool
+	projectDir        string
 	verbose           bool
-	debug             bool
-)
-
-var (
-	configCommandString = []string{"config"}
-	globalConfigKeys    = []string{"airflow_home", "airflow_dags_folder"}
 )
 
-func getAbsolutePath(path string) (string, error) {
-	if !filepath.IsAbs(path) || path == "" || path == "." {
-		currentDir, err := os.Getwd()
-		if err != nil {
-			err = fmt.Errorf("error getting current directory %w", err)
-			return "", err
-		}
-		path = filepath.Join(currentDir, path)
-	}
-	return path, nil
+// Build the cmd string to execute
+func buildCmd(cmd *cobra.Command, args []string) []string {
+	cmdArgs := initCmdArgs(cmd, args)
+	cmdArgs = extendCmdArgs(cmd, cmdArgs)
+	return append(append(initArgs(), cmd.Name()), cmdArgs...)
```

Comment:
the function names here are confusing. `initCmdArgs` can be read as args for the init command because we also have a variable `initCmdName` so someone might relate `initCmdArgs` to init command. 
Also, there is `initCmdArgs` and `initArgs` which both initialise args, can we rename them to something  more descriptive based on their differences?

Reply:
Yes good catch!

Reply:
Fixed in db57516